modules/lang-painless/src/main/java/org/elasticsearch/painless/node/SExpression.java

```diff
@@ -44,36 +44,40 @@ public SExpression(Location location, AExpression expression) {
     }
 
     @Override
-    void analyze(ScriptRoot scriptRoot, Scope scope) {
+    Output analyze(ScriptRoot scriptRoot, Scope scope, Input input) {
+        this.input = input;
+        output = new Output();
```

Comment:
Consider creating output close to it's first use, at line 68.

Reply:
Done.